apps/web-giddh/src/app/user-details/user-details.component.html

```diff
@@ -356,14 +352,7 @@ <h1 class="section-head pt-0">{{ localeData?.session?.section_heading }}</h1>
                     </div>
                 </div>
             </mat-tab>
-            <mat-tab *ngIf="selectedCompany?.planVersion === 1 && selectedCompany?.subscription?.status !== 'expired'"
```

Comment:
Need confirmation on this removal from kriti

Reply:
Done Need add this code